### ISSUE TITLE
Unsupported SET option client_min_messages

### ISSUE THREAD

**Issue by pietersv** [plotly/falcon-sql-client#420]
Falcon looks amazing.  Attempting to connect to a Vertica instance using the PostgresQL connection type.  It almost works but fails on an unsupported option:

`Unsupported SET option client_min_messages`

Is Vertica support in the dev pipeline?  Alternatively is it possible to note and proceed when some options aren't set?  Usually Postgres drivers work fine with Vertica but not all newer Postgres options are supported.

**Comment by n-riesco** [comment]
We've already opened issue #344 to track the development of a connector for Vertica. As far as I know, no one has started to work on this issue.

I've quickly looked into the possibility of using the PostgresQL connector to connect to Vertica. And it seems this isn't possible without developing a new dialect for `sequilize`. There was an attempt [here](https://github.com/sequelize/sequelize/pull/7269), but it's now closed.

As I mentioned in #344, [node-vertica](https://github.com/wvanbergen/node-vertica), although unmaintained, seems a feasible option.

---

Closing in favour of #344
Writing a connector based on `node-vertica` is a good first-issue to contribute to Falcon. If anyone steps in, we (the maintainers) will help.
